### ISSUE TITLE
Invalid YAMCS_PROCESSOR value when running script from web interface

### ISSUE THREAD

**Issue by fgretief** [yamcs/yamcs#882]
The Yamcs web interface for running a script (Procedures -> Run a script) is setting the YAMCS_PROCESSOR value incorrectly.

I started with a clean [quickstart](https://github.com/yamcs/quickstart) project and created a small script that prints out the Yamcs environment variables.  The output of my first try looked like this:
```
Starting activity
Started process, pid=19017
 YAMCS_URL: http://0.0.0.0:8090
 YAMCS_API_KEY: None
 YAMCS_INSTANCE: myproject
 YAMCS_PROCESSOR: myproject
Process has terminated
Activity successful
```
The `YAMCS_PROCESSOR` variable was set to `myproject`, but I expected it to be `realtime`.

In another script I used the [Yamcs Client](https://docs.yamcs.org/python-yamcs-client/) to issue a command. The script  `battery.py` looked like this:
```python
#!/usr/bin/env -S python3 -u
import os
from yamcs.client import YamcsClient
client = YamcsClient.from_environment()
processor = client.get_processor(os.environ["YAMCS_INSTANCE"],
                                 os.environ["YAMCS_PROCESSOR"])
processor.issue_command("/myproject/SwitchVoltageOn", args={"Battery":"1"})
```

This resulted in the following error when it was triggered from the web interface:
```
Starting activity
Started process, pid=22205
Traceback (most recent call last):
  File "/home/user/quickstart/target/yamcs/etc/scripts/battery.py", line 7, in <module>
    processor.issue_command("/myproject/SwitchVoltageOn", args={"Battery":"1"})
  File "/home/user/quickstart/.venv/lib/python3.10/site-packages/yamcs/tmtc/client.py", line 766, in issue_command
    response = self.ctx.post_proto(url, data=req.SerializeToString())
  File "/home/user/quickstart/.venv/lib/python3.10/site-packages/yamcs/core/context.py", line 87, in post_proto
    return self.request("post", path, **kwargs)
  File "/home/user/quickstart/.venv/lib/python3.10/site-packages/yamcs/core/context.py", line 115, in request
    raise NotFound(f"404 Client Error: {msg}")
yamcs.core.exceptions.NotFound: 404 Client Error: No processor 'myproject' within instance 'myproject'
Process returned with exit value 1
Activity failed: Process returned with exit value 1
```
![image](https://github.com/yamcs/yamcs/assets/1013871/c2fb5f63-1d93-454e-8150-1bfc3a91069e)

Then I tried the following script to trigger the `battery.py` script:
```python
#!/usr/bin/env -S python3 -u
from yamcs.client import YamcsClient
client = YamcsClient("http://localhost:8090/")
processor = client.get_processor("myproject", "realtime")
processor.run_script("battery.py")
```

Which resulted in the successful execution of the `battery.py` script.
![image](https://github.com/yamcs/yamcs/assets/1013871/43e6755f-e8d8-41ad-9543-73ee4eb82b97)

Is there an configuration setting that I missed or is this an issue with the web interface?

**Comment by fqqb** [comment]
No you're right, it's an issue with the Web UI.
It's addressed in the pending PR #881

**Comment by fgretief** [comment]
Thanks
